dashboard/app/models/pd/application/teacher1920_application.rb

```diff
@@ -146,7 +146,7 @@ def principal_approval_state
       response = Pd::Application::PrincipalApproval1920Application.find_by(application_guid: application_guid)
       return COMPLETE + response.full_answers[:do_you_approve] if response
 
-      principal_approval_email = emails.find_by(email_type: 'principal_approval')
+      principal_approval_email = emails.where(email_type: 'principal_approval').last
```

Comment:
Do we need to enforce a particular order here, or is it implied that the most recently inserted row is always the right one?

Reply:
Good point, and something I also wondered.  `.last` [defaults](https://apidock.com/rails/ActiveRecord/FinderMethods/last) to primary key, which I presumed would be reliable because I'm pretty sure we only add new emails to an application, but we could sort by something like `created_at` if you or @clareconstantine think that's better.

Reply:
`created_at` seems like the most reliable thing, but I think you're right jn that the most recently created will have the highest primary key (we never transfer emails between applications, we always create new ones)